hatrac/model/storage/amazons3.py

```diff
@@ -10,80 +9,65 @@
 object-version lifecycle and authorization is handled by the caller.
 
 """
-import os
 import tempfile
 from webauthn2.util import PooledConnection
-import boto
-import boto.s3
-import boto.s3.key
-from boto.s3.connection import S3Connection
-from boto.exception import S3ResponseError
-from hatrac.core import BadRequest, coalesce
+import boto3
+from botocore.exceptions import ClientError
+from hatrac.core import NotFound, BadRequest, coalesce
 import binascii
 import base64
+import logging
```

Comment:
Please do not add another logging facility.   We already have web.debug() for basic debug tracing and hatrac.rest.core.request_trace() to add stuff to the syslog-based audit log.

Reply:
Fixed to request_trace() where needed.